### DISCUSSION TITLE
WARNI [open_webui.main] Frontend build directory not found at 'C:\Users\..\open-webui\backend\open_webui\frontend'. Serving API only.

### DISCUSSION THREAD

**Discussion by RamS1005** [open-webui/open-webui#8009] [Issues]
Dear community members, I have this issue and unable to render the UI. I have installed it using pip and not with docker approach.

**Comment by TheMcSebi** [comment]
I have the same issue, I tried to fix it using "npm run build" but I wasn't successful yet.

Edit T+2min: simply renaming the "build" folder that get's created (or maybe it was there before?) when running "npm run build" to "frontend", and moving it to the "backend/open_webui" folder did the job

**Reply by RamS1005** [reply]
Hello there, yes. It worked this way. I had renamed the folder as "frontend" and ran build. Works and I can see the GUI. Nice stuff.

**Comment by surak** [comment]
I made it as a symbolic link, but this is wrong. The directory should be there or be found properly.
